<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYControl</class>
 <widget class="QWidget" name="XYControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5" rowspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QTextEdit" name="textEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>升降调</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>打开文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="pushButton_6">
     <property name="text">
      <string>停止播放</string>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>X电机</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QPushButton" name="pushButton_7">
     <property name="text">
      <string>打开midi</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="4">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>清除</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>歌曲节拍（拍/分钟）</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>音符</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="text">
      <string>120.0</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>音长</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Y电机</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_5">
     <property name="text">
      <string>测试</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>音高</string>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="3">
    <widget class="QSlider" name="horizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>保存为文件</string>
     </property>
    </widget>
   </item>
   <item row="5" column="9">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="8">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
